dms/models/directory.py

```diff
@@ -510,6 +502,12 @@ def _onchange_category(self):
             )
             record.tag_ids = tags
 
+    @api.onchange("storage_id")
+    def _onchange_storage_id(self):
+        for record in self:
+            if record.storage_id.save_type == "attachment":
+                record.group_ids = None
```

Comment:
In Odoo, the emptiness is represented as `False` usually:

```suggestion
                record.group_ids = False
```